frontend/before-tests.js

```diff
@@ -3,12 +3,7 @@
 import { configure } from 'enzyme';
 import * as Adapter from 'enzyme-adapter-react-16';
 
-import './setup-jsdom';
```

Comment:
I've realized that Jest takes care of this already via [`testEnvironment`](https://jestjs.io/docs/en/configuration.html#testenvironment-string) option which defaults to `jsdom`, i.e. using `jest-environment-jsdom` implementation.

Jest `TestEnvironment` is sandboxed, with each test suite having its own env. instance prepared upon setup and destroyed upon teardown.

Since `plugin-stats` now uses [browser-env](https://github.com/lukechilds/browser-env) to setup `jsdom`, I'm removing this import.